docs/tethys_portal/configuration.rst

```diff
@@ -0,0 +1,372 @@
+.. _tethys_configuration:
+
+***************************
+Tethys Portal Configuration
+***************************
+
+Beginning in Tethys Platform 3.0 the Tethys Portal is configured via a :file:`portal.yml` file in the :file:`$TETHYS_HOME/` directory. This instructions outline the various settings that can be added to the :file:`portal.yml` file.
+
+Once you have installed Tethys you can generate a new :file:`portal.yml` file using the ``gen`` command. (See :ref:`tethys_gen_cmd` for more information).
+
+::
+
+  tethys gen portal
+
+This will create a new :file:`portal.yml` file in you ``TETHYS_HOME`` directory that looks like this:
+
+.. code-block:: yaml
+
+    # Portal Level Config File
+
+    # see tethys documentation for how to setup this file
+    version: 1.0
+    name: test
+    apps:
+    settings:
+      SECRETE_KEY: ...
+    site_content:
+
+
+
+You can now customize this file either by manually editing it, or by using the :ref:`tethys_settings_cmd`.
+
+.. caution::
+
+  The :ref:`tethys_settings_cmd` will rewrite the :file:`portal.yml` file each time it is run and will not preserve comments.
+
+Portal Yaml Keys
+----------------
+
+The following is a list of keys that can be added to the :file:`portal.yml` file:
+
+* ``version``: The version of the :file:`portal.yml` file schema
+* ``name``:
+* ``apps``:
+* ``settings``:
+
+  * ``SECRET_KEY``:
```

Comment:
Are you planning to add descriptions for these? I'd be happy to help out here if you just want to get it merged.

Reply:
Yes, I could definitely use some help! I meant to add descriptions to each key, but I'm now way overdue to be done with this and have some other deadlines that I need to scramble to hit.

Reply:
Let’s go ahead and merge for now.